Routers/BookingRouter.cs

```diff
@@ -0,0 +1,118 @@
+﻿using AirportTicketBooking.Classes;
+using AirportTicketBooking.Repositories;
+using ClassType = TicketClassType.ClassType;
+
+namespace AirportTicketBooking.Routers
+{
+    internal class BookingRouter
+    {
+        private BookingRepositoty _bookingRepositoty;
+        private FlightRouter _flightRouter;
+
+        public BookingRouter()
+        {
+            _bookingRepositoty = new BookingRepositoty();
+        }
+
+        public List<Booking> GetAllData()
+        {
+            return _bookingRepositoty.Load();
+        }
```

Comment:
I might understand that this method will be limited to this class to load the data and then we can use its result to filter the data based on the request. In this case, it should be a private method. 

Note: this should be applied to any other method that will be a helper method specific to this class. I don't know which method will be like that since I don't know how you are thinking about implementing the rest of your app.